app/services/metrics/calculator.rb

```diff
@@ -21,17 +21,36 @@ def escorts_by_date
     end
 
     def hours_saved
-      hours = (total_reused_escorts * 23.minutes) / (60 * 60)
+      hours = (total_reused_escorts * 23.minutes) / 3600
```

Comment:
I always like 'magic numbers' like these to be held in constants e.g. `AVG_PER_REUSE_MINS = 23`